service/sg-windows/sg-accel-service/sg-accel-service.go

```diff
@@ -76,26 +76,37 @@ func main() {
 
 	var exePath string
 	var configPath string
+	var stderrPath string
 
 	switch len(os.Args) {
 	case 2:
 		exePath = "C:\\Program Files (x86)\\Couchbase\\sg_accel.exe" // Uses default binary image path
-		svcConfig.Arguments = []string{"start"}                      // Uses the default config
+		stderrPath = "C:\\Program Files (x86)\\Couchbase\\var\\lib\\couchbase\\logs\\sg_accel_error.log" // Uses default stderr path
+		svcConfig.Arguments = []string{"start", stderrPath}                      // Uses the default config
```

Comment:
`stderrPath` should be pulled out into a single constant rather than being duplicated several times.